src/utils/src/RSAnalysisEngine.cpp

```diff
@@ -41,11 +41,6 @@ RSAnalysisEngine::~RSAnalysisEngine()
     delete engine;
     engine = NULL;
   }
-  if(rspm)
-  {
-    delete rspm;
-    rspm = NULL;
-  }
 }
```

Comment:
@anindex  did this have anything to do with why your unit test were failing?

Reply:
I just remove the redundant code, cause it is a duplication. And it also has no effect on the failing, so if it causes some problem I will put it back

Reply:
oh, sorry, I did not notice that a few lines above it is already deleted :)